packages/bot-web-ui/src/assets/sass/run-panel.scss

```diff
@@ -66,3 +66,43 @@
         }
     }
 }
+
+.self-exclusion {
+    font-size: var(--text-size-xs);
+    font-weight: normal;
+    line-height: 1.43;
+    color: var(--text-general);
+    margin: 1.6rem;
+
+    &__dialog {
+        max-height: 500px !important;
```

Comment:
Can you please add TODO here.

Reply:
@sam-binary  I don't get this comment :)

Reply:
you're using `!important` because the css from bot is not overriding css from core right? Please add this `// TODO: [fix-dc-bundle] Fix import issue with Deriv Component stylesheets (app should take precedence, and not repeat)` so that when core team fix it, all these `!important` can be remove